### DISCUSSION TITLE
groups / bookmark change in v0.3.0

### DISCUSSION THREAD

**Discussion by airtonix** [toboshii/hajimari#72] [Q&A] [answered]
https://github.com/toboshii/hajimari/releases/tag/v0.3.0 reads: 

> groups has been deprecated and is now bookmarks (or globalBookmarks in config.yaml), a nested list of [BookmarkGroups](https://github.com/toboshii/hajimari#bookmarkgroup) and [Bookmarks](https://github.com/toboshii/hajimari#bookmark)

But there's no mention of how the `hajimari.io/group` annotation will be handled.

**Comment by toboshii** [comment, marked as answer]
There is no relationship between `hajimari.io/group` and bookmarks which is one of the reasons I renamed the bookmark configuration option.

By default applications are grouped by their namespace (if you enable `showAppGroups`). `hajimari.io/group` allows you to change this behavior and specify the apps group yourself, for example of you had all your apps deployed in the default namespace.
